### ISSUE TITLE
Use marker files to signify which directories to exclude

### ISSUE THREAD

**Issue by effulgentsia** [php-tuf/staged-composer-update#8]
Currently, $excludeDirectory is [hard-coded](php-tuf/staged-composer-update/blob/main/src/StagedUpdateCommand.php#L22).

First of all, there can be multiple directories that need to be excluded, so this should be an array.

Secondly, @davidstrauss had the idea to let the directories mark themselves, by e.g., having an empty `.tufignore` file in them. The code could then build up the list of exclude directories by finding all the ones with this empty file.

Not sure if `.tufignore` is a great name choice. Any ideas for a better name?

**Comment by davidstrauss** [comment]
> Not sure if .tufignore is a great name choice. Any ideas for a better name?

Agreed. This functionality is arguably more related to the staged updater plug-in for Composer than anything for TUF. It should be possible to use this plugin without TUF at all. Maybe something referring to this plugin, then?

**Comment by effulgentsia** [comment]
How about `.composer-staged-update-exclude` or `.csuignore`?
